README.md

```diff
@@ -144,6 +153,30 @@ There are two parameters:
 * a space number (it could just as easily have been a space name), and
 * a tuple.
 
+### msgpack.v5 migration
+
+Most function names and argument types in `msgpack.v5` and `msgpack.v2`
+have not changed (in our code, we noticed changes in `EncodeInt`, `EncodeUint`
+and `RegisterExt`). But there are a lot of changes in a logic of encoding and
+decoding. On the plus side the migration seems easy, but on the minus side you
+need to be very careful.
+
+First of all, `EncodeInt8`, `EncodeInt16`, `EncodeInt32`, `EncodeInt64`
+and `EncodeUint*` analogues at `msgpack.v5` encode numbers as is without loss of
+type. In `msgpack.v2` the type of a number is reduced to a value.
+
+Secondly, a base decoding function does not convert numbers to `int64` or
+`uint64`. The change makes manual type conversions much more difficult and can
```

Comment:
It's not clear from that description what type v5 convert numbers to.

Reply:
I tried to make it more clear:
```patch
--- a/README.md
+++ b/README.md
@@ -166,14 +166,15 @@ and `EncodeUint*` analogues at `msgpack.v5` encode numbers as is without loss of
 type. In `msgpack.v2` the type of a number is reduced to a value.
 
 Secondly, a base decoding function does not convert numbers to `int64` or
-`uint64`. The change makes manual type conversions much more difficult and can
-lead to runtime erros with an old code. We recommend not use type conversions
+`uint64`. It converts numbers to an exact type defined by MessagePack. The
+change makes manual type conversions much more difficult and can lead to
+runtime errors with an old code. We do not recommend to use type conversions
 and give preference to `*Typed` functions (besides, it's faster).
```